stats/aggregated_stats.go

```diff
@@ -49,10 +50,19 @@ func convertCadvisorStatToAggregatedStat(id string, name string, containerIds ma
 		if idVal, ok := containerIds[id]; ok {
 			id = idVal
 		} else {
-			return AggregatedStat{}
+			found := false
+			for _, alias := range aliases {
+				if idVal, ok := containerIds[alias]; ok {
+					id = idVal
+					found = true
```

Comment:
Its just good practice to break out of a for loop once you find what you are looking for. Can you please change it?